### DISCUSSION TITLE
Wrong content redirection issue for collaborator and content creator.

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#716] [Bugs]
Hello, we're using the Sunbird 5.0 version.
Whenever a collaborator adds multiple pieces of content to the same course unit, the material is incorrectly misdirected for both the collaborator and the content author.

A collaborator adding a unit to the course and adding multiple pieces of content within that unit will cause the unit to redirect to another piece of content when the collaborator clicks on it.
And this is taking place for both content creators and collaborators. It functions well for the remaining users.
For reference, kindly watch the attached videos.
Kindly provide a fix or workaround for the same.

https://github.com/Sunbird-Ed/Community/assets/109504832/8a606630-668f-4a5f-84d2-268632ae03e6

**Comment by anisha147** [comment]
Hi @sachinb80 

I request you to move this discussion to the knowledge building block's discussion forum.

Thanks!

**Reply by sachinb80** [reply]
Okay @anisha147
